public/index.html

```diff
@@ -9,6 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
```

Comment:
I think we need to harmonize how we install and configure tailwinds CSS. If to use CDN or run the configurations.

Reply:
We will definitely go with the configuration :)